.github/workflows/build.yml

```diff
@@ -5,19 +5,19 @@ on:
       RELEASE:
         description: 'AlmaLinux release'
         required: true
-        default: '9'
+        default: '10'
         type: choice
         options:
-          - 10-kitten
-          - 10
-          - 9
-
+          - '10-kitten'
+          - '10'
+          - '9'
   schedule:
     # run every day at 03:00 UTC
     - cron:  '00 03 * * *'
 
+
 env:
-  LATEST_MAJOR: 9
+  LATEST_MAJOR: 10
```

Comment:
I think it's too early for this, it should be coordinated with a similar change for the standard docker images and it needs to be communicated properly.